### PR TITLE
feat: adpoting mmt part 2

### PR DESCRIPTION
**Pull request by mwdchang**
### Summary
Adopt the notions of multiple (subject-outcome, subject-controllers, outcome-controllers) matrices for doing stratified-model parameterization.

As well, this PR removes the notion of stratification-type, rather than mira/catlab/null, a model is either stratified or not-stratified. Additional cleanups are done to remove unused model-representation code.

### CODE REVIEW

File: packages/client/hmi-client/src/utils/easter-eggs.ts
Comment (YohannParis) [inline packages/client/hmi-client/src/utils/easter-eggs.ts:1]: Thank you for doing this, I was too lazy to do so.

File: packages/client/hmi-client/src/components/model/petrinet/model-configurations/tera-stratified-matrix.vue [hunk 1]
```diff
@@ -333,6 +313,7 @@ onMounted(() => {
 
 .p-dropdown {
 	min-width: 11rem;
+	margin-bottom: 1rem;
```
Comment (YohannParis) [inline packages/client/hmi-client/src/components/model/petrinet/model-configurations/tera-stratified-matrix.vue:316]:
```suggestion
	margin-bottom: var(--gap);
```

File: packages/client/hmi-client/src/workflow/ops/model-config/tera-initial-table.vue [hunk 2]
```diff
@@ -219,11 +220,13 @@ const typeOptions = [
...
 const props = defineProps<{
 	modelConfiguration: ModelConfiguration;
+	mmt: MiraModel;
+	mmtParams: MiraTemplateParams;
 	data?: ModelConfigTableData[];
 	hideHeader?: boolean;
 }>();
 
-const emit = defineEmits(['update-value', 'update-configuration']);
+const emit = defineEmits(['update-value', 'update-configuration', 'update-cell-value']);
```
Comment (blanchco) [inline packages/client/hmi-client/src/workflow/ops/model-config/tera-initial-table.vue:229]: i dont believe 'update-cell-value' is emitted in this component

**Review by blanchco — APPROVED** [review]
looking good 👍

**Review by YohannParis — APPROVED** [review]

**Pull request merged**